### PR TITLE
Normalize whitespace in sites.json

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

**Comment by bsipocz** [comment]
Merging #60 causing conflicts. Could you please rebase?

File: coordinates/sites.json [hunk 1]
```diff
@@ -753,51 +753,55 @@
...
+        "latitude": 34.2583,
+        "latitude_unit": "degree",
+        "longitude": 243.0786,
+        "longitude_unit": "degree",
+        "timezone": "US/Pacific",
+        "aliases": [
+            "BBSO",
+            "NST"
+        ],
+        "source": "BBSO website: http://www.bbso.njit.edu/newinfo.html",
```
Comment (dhomeier) [inline coordinates/sites.json]:
```suggestion
        "source": "BBSO website: http://www.bbso.njit.edu/newinfo.html"
```
Typo in rebase

**Comment by dhomeier** [comment]
Thanks for the cleanup; I've rebased and fixed some additional tab-usage in the base branch.

**Review by hamogu — APPROVED** [review]
Looks good, 
```
In [1]: from astropy.coordinates.tests import test_sites 
   ...: test_sites.check_builtin_matches_remote('coordinates/sites.json') 
```
passes. 
Thanks for cleaning up!

**Pull request merged**